Source/UI/NewUI/CommonUI/View/TextFieldView.swift

```diff
@@ -81,6 +90,13 @@ class TextFieldView: UIView {
         headerLabel?.textColor = style.title?.textColor
     }
 
+    private func setupMandatoryLabel(style: CellTextFieldStyle) {
+        mandatoryLabel?.isHidden = style.isMandatory
+        mandatoryLabel?.text = Constants.LocalizationKeys.BillingForm.Cell.optionalInput
+        mandatoryLabel?.font = UIFont(graphikStyle: .regular, size:  Constants.Style.BillingForm.InputOptionalLabel.fontSize.rawValue)
+        mandatoryLabel?.textColor = .doveGray
+    }
```

Comment:
Sorry, I think you misunderstood my previous suggestion. I was not saying the method should be renamed from `update` to `setup`, I was suggesting that 3 out of these 4 lines should be moved into the existing `setupMandatoryLabel`, and only the visibility should be updated (which may even remove the need for this method altogether.

Also I wouldn't want to have 2 methods named identically, once taking a parameter, one without. `function overloading` definitely is a useful feature of Swift, just not sure this is a good use case for it.